### PR TITLE
Updates information for K-9

### PR DESCRIPTION
**Pull request by reitzig**
K-9 Mail supports Autocrypt since release [5.400](https://github.com/k9mail/k-9/releases/tag/5.400).

Also, fixes a repository-internal deadlink.

### CODE REVIEW

**Comment by Valodim** [comment]
Autocrypt support is fairly experimental in K-9: The key choice algorithm is not up to date with the 1.0 spec, and it's missing gossip and autocrypt setup messages. Autocrypt is *not* just "putting keys in headers", so I'm a little uncomfortable announcing "Autocrypt support" without qualification like that.

**Comment by reitzig** [comment]
That's too bad. I took the release notes on face value. My bad for not checking more closely.

**Comment by Valodim** [comment]
Sorry about that :) It's hard to find a good granularity of information to give to users.

An update to actual 1.0 logic and gossip is in the [pipe](https://github.com/open-keychain/open-keychain/pull/2253) [line](https://github.com/k9mail/k-9/pull/3105) though. Autocrypt Setup Message will follow.

File: doc/install.rst [hunk 1]
```diff
@@ -26,16 +26,16 @@ You can test it via `Nightly Build`_ today.
 
 .. _Enigmail: https://enigmail.net
 .. _Thunderbird: https://www.mozilla.org/en-US/thunderbird/
-.. _`Nightly Build`: install-autocrypt-enigmail.html
+.. _`Nightly Build`: install-autocrypt-enigmail.rst
```
Comment (r10s) [inline doc/install.rst]: Offtopic, however, as we want the link to work on the rendered homepage, the existing `.html` link is correct  (the rendered homepage has HTML-files, not reStructuredText-files). I do not know a solution that works on both, the rendered homepage and on the preview on github.

**Pull request closed without merging**